scp/lib/ui/views/mentor_search/mentor_page.dart

```diff
@@ -334,11 +334,13 @@ class MentorDetails extends StatelessWidget {
   }
 
   void saveContact(Contact contact) async {
-    // save contact
-    final result = FlutterContacts.insertContact(contact).then((value) {
-      print('add contact dialog closed');
-    }).catchError((error) {
-      print('pata nai bhai kya error hai');
-    });
+    if (await FlutterContacts.requestPermission()) {
+      // save contact
+      final result = FlutterContacts.insertContact(contact).then((value) {
+        print('add contact dialog closed');
+      }).catchError((error) {
+        print('pata nai bhai kya error hai');
```

Comment:
Instead of just printing, can we show some snackbar or any other notification widget to show user something went wrong? Or Unable to save / do the action?

Reply:
yes,we can show the snackbar same as one used in no internet connection with text "you need permission to create the contact"

Reply:
preview:
![Screenshot_2022-01-12-21-40-54-249_in ac nitrkl scp scp](https://user-images.githubusercontent.com/74104690/149180250-a435ec1e-f053-4077-a4db-1fc73ffa3cf3.jpg)